src/components/Navigation/TopBar/Help/ReleaseNotesDialog/ReleaseNotesHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+
+import { Typography } from '@equinor/eds-core-react';
+
+import FilterHeader from './Filters/FilterHeader';
+import {
+  FilterContainer,
+  HeaderWrapper,
+  HeadingContainer,
+} from './ReleaseNotesHeader.styles';
+
+const ReleaseNotesHeader: FC = () => {
+  return (
+    <HeaderWrapper>
+      <HeadingContainer className="release-notes-header-container">
+        <Typography group="heading" variant="h4">
```

Comment:
Don't have to change this but IIRC you don't need 'group="heading"' here

Reply:
tested and confirmed; removed